nixos/modules/virtualisation/virtualbox-guest.nix

```diff
@@ -27,6 +49,33 @@ in
       type = types.bool;
       description = lib.mdDoc "Whether to enable x11 graphics";
     };
+
+    conditionVirtualization = mkOption {
+      default = "oracle";
+      type = types.str;
+      description = lib.mdDoc ''
+        The virtualized environment in which the guest additions should be started.
+        E.g., "oracle" or "kvm"
+      '';
+    };
+
+    clipboard = mkOption {
+      default = true;
+      type = types.bool;
+      description = lib.mdDoc "Whether to enable clipboard support";
+    };
+
+    seamless = mkOption {
+      default = true;
+      type = types.bool;
+      description = lib.mdDoc "Whether to enable seamless support";
```

Comment:
Nit: really vague description here, what does this actually do?

Reply:
This option allows seamless integration of guest application windows into the host window system. Haven't used this in a while and I'm not sure how this copes in a Wayland world.

Reply:
Yeah we should definitely note that - I'm assuming it doesn't work on Wayland, unless the guest side does some portal shenanigans?

Reply:
As far as I am aware, it is always started when installing the guest additions on other systems using the iso installer. Hence, I decided to start it by default as well.

Reply:
Yeah, I meant the guest. I'm not saying we shouldn't enable this btw - just saying it could use a better description.